doc-deploy-dev/versions.js

```diff
@@ -0,0 +1,41 @@
+function updateContent() {
+  Promise.all([
+    fetch("index.html")
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error(`HTTP error: ${response.status}`);
+        }
+        return response.text();
+      })
+      .then((html) => {
+        document.documentElement.innerHTML = html;
+      }),
+    fetch("versions.json")
+      .then((res) => res.json())
+      .then((data) => {
+        var table = "<h2>Older versions</h2>";
+        table += "<table class='table'>";
+        table += "<th> versions </th>";
+        table += "<th> url </th>";
+        for (let versions of data) {
```

Comment:
Let's play also with a variable named `ith_version` so we can add the `row-odd` and `row-even` classes to the right rows in the table. This guarantees that the final style of the table matches the one from the ansys-sphinx-theme.